sdl_android_lib/src/com/smartdevicelink/proxy/rpc/DateTime.java

```diff
@@ -0,0 +1,132 @@
+package com.smartdevicelink.proxy.rpc;
+
+import com.smartdevicelink.proxy.RPCStruct;
+
+import java.util.Hashtable;
+
+public class DateTime extends RPCStruct{
+    public static final String KEY_MILLISECOND = "millisecond";
+    public static final String KEY_SECOND = "second";
+    public static final String KEY_MINUTE = "minute";
+    public static final String KEY_HOUR = "hour";
+    public static final String KEY_DAY = "day";
+    public static final String KEY_MONTH = "month";
+    public static final String KEY_YEAR = "year";
+    public static final String KEY_TZ_HOUR = "tz_hour";
```

Comment:
Can you write documentation to explain these (tz) variables and methods a little more? And what's the purpose of having both tz hours and normal hours?